README.md

```diff
@@ -9,6 +9,8 @@ A simple web page to hand off users to the Microsoft Health bot
 
 [![Deploy to Azure](https://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)
 
+Note: to enable online file editing in the portal please choose "Windows", otherwise we recommend you to go with the default (Linux)
```

Comment:
@GuyBeckerMicrosoft do you have any comments on the wording? maybe using the teem "Azure Portal" instead of "portal", to not be confused with the bot portal